src/mcts/params.cc

```diff
@@ -54,6 +54,9 @@ const OptionId SearchParams::kCpuctId{
     "cpuct_init constant from \"UCT search\" algorithm. Higher values promote "
     "more exploration/wider search, lower values promote more "
     "confidence/deeper search."};
+const OptionId SearchParams::kCpuctAtRootId{
+  "cput-at-root", "CpuctAtRoot",
```

Comment:
There's a mistype in flag name, and also changes doesn't seem to be formatted (`clang-format -style=google` can do that for you)